### PR TITLE
Update installer

### PR DESCRIPTION
**Pull request by AndrewSazonov**
- [x] Add licence information to the installer
- [x] Check if sudo is really needed for installer

### CODE REVIEW

File: Scripts/InstallPyPackages.py [hunk 1]
```diff
@@ -80,16 +80,17 @@ def fixDictdifferNumpy():
 
     upgradePip()
 
-    installFromGit(owner='ikibalin', repo='cryspy', branch='transition-to-version-0.2')
+    #installFromGit(owner='pyinstaller', repo='pyinstaller', branch='develop')
```
Comment (wardsimon) [inline Scripts/InstallPyPackages.py:83]: This commented code is not needed

**Comment by wardsimon** [comment]
And version info:
Version info

File: Scripts/InstallPyPackages.py (hunk 1 above)
Comment (AndrewSazonov) [reply Scripts/InstallPyPackages.py:83]: Just kept it to simplify switching from the master branch to another one for the cryspy library, when needed.

**Comment by AndrewSazonov** [comment]
I have created issues #60, #61 and #62 to fix code singing, app version and installer styling, respectively.

**Pull request merged**